src/core/src/op/result.cpp

```diff
@@ -49,9 +49,7 @@ bool Result::evaluate(TensorVector& outputs, const TensorVector& inputs) const {
     }
 
     outputs[0].set_shape(inputs[0].get_shape());
-    void* output = outputs[0].data();
-    const void* input = inputs[0].data();
-    memcpy(output, input, outputs[0].get_byte_size());
+    inputs[0].copy_to(outputs[0]);
```

Comment:
We need to tune performance of `copy_to` then if we worry about it. We need to use `copy_to` ideally here because this method is specially created for it.
Now I agree with you to avoid performance degradation.

Reply:
Fixed.